modules/planning/proto/st_bounds_decider_config.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto2";
+
+package apollo.planning;
+
+message STBoundsDeciderConfig {
+  optional double total_time = 1 [default = 7.0];
```

Comment:
Seems like `float` is enough.

Reply:
Yes, that's enough. But since all other places used double for time, I just wanted to be consistent. Anyway we are not that scarce in memory.